svo (install, config, pipes, rift, parry, prios).xml

```diff
@@ -3963,11 +3963,6 @@ end
 
 do
   local oldCL = createLabel
```

Comment:
Do we still need this line if we aren't overwriting the function?

Reply:
Hmmm I don't know.... there is a 'do' there and I have no idea where the statement for it starts and I'm a bit afraid of removing it and breaking the whole structure '-'